### ISSUE TITLE
Grid artifacts cause?

### ISSUE THREAD

**Issue by rahimr2** [chiahao3/ptycho#1]
Hi,

I was wondering if you were familiar with this artifact/failure in ptyrad:

<img width="742" height="744" alt="Image" src="https://github.com/user-attachments/assets/c1b3624f-8c2f-4612-9089-0806f20150c6" />

Specifically the grid-like periodic texture in the image. 

Looking at the slices, the first slice in the stack seems promising:

<img width="742" height="743" alt="Image" src="https://github.com/user-attachments/assets/5c0de16e-be66-4ec3-a23a-1b628f78d673" />

but then the rest of the slices in the stack become background fluctuation with increasingly prominent grid texture:

<img width="743" height="743" alt="Image" src="https://github.com/user-attachments/assets/e4f8d2df-2345-45bf-b5ea-2377f3efc69c" />

I was wondering if you happen to know what causes this? I am seeing this in a couple of my reconstructions, and sometime it goes away but sometimes it doesn't. So far I have tried adjusting slices, slice thickness, batch size, and probe learning rate. 

My current hypothesis is that it arises from a probe that is too constrained. (I lowered its lr to avoid probe object mixing. Perhaps there is a better way to avoid it, though, like hyperparameter tuning to find better probe parameters?).

Do you have thoughts on this? I am happy to provide more information on the parameters/dataset if that helps. I started with the PSO .yml file and made the appropriate adjustments to it.
